contracts/utils/test/safe_math.ts

```diff
@@ -0,0 +1,171 @@
+import { chaiSetup, constants, provider, txDefaults, web3Wrapper } from '@0x/contracts-test-utils';
+import { BlockchainLifecycle } from '@0x/dev-utils';
+import { BigNumber, SafeMathRevertErrors } from '@0x/utils';
+import * as chai from 'chai';
+import * as _ from 'lodash';
+
+import { artifacts, TestSafeMathContract } from '../src';
+
+chaiSetup.configure();
+const expect = chai.expect;
+const blockchainLifecycle = new BlockchainLifecycle(web3Wrapper);
+
+function toBN(a: number | string): BigNumber {
```

Comment:
Can we rename this `toBigNumber`? `BN` is the name of another popular BigNumber library.